lib/saml/url_service.rb

```diff
@@ -4,26 +4,26 @@ module SAML
   # This module is responsible for providing the URLs for the various SSO and SLO endpoints
   module URLService
     # SSO URLS
-    def mhv_url
-      build_sso_url(authn_context: 'myhealthevet', connect: 'myhealthevet')
+    def mhv_url(alt_relay: false)
+      build_sso_url(authn_context: 'myhealthevet', connect: 'myhealthevet', alt_relay: alt_relay)
     end
 
-    def dslogon_url
-      build_sso_url(authn_context: 'dslogon', connect: 'dslogon')
+    def dslogon_url(alt_relay: false)
+      build_sso_url(authn_context: 'dslogon', connect: 'dslogon', alt_relay: alt_relay)
     end
 
-    def idme_loa1_url
-      build_sso_url
+    def idme_loa1_url(alt_relay: false)
```

Comment:
if these urls are environment variable specific, why is there a need for the argument? Shouldn't it be based on the current environment variable?

Reply:
> if these urls are environment variable specific

Are you assuming that each `vets.gov` & `va.gov` will have their own `vets-api` instances? AFAIK this is not a valid assumption. One `vets-api` instance may serve requests coming from either `vets.gov` or `va.gov`.  

Now that I think of it though, it may also be serving requests to `preview.va.gov`...